### ISSUE TITLE
Could not find any code.

### ISSUE THREAD

**Issue by HenkySerpent** [eclipse-tractusx/pcf-exchange-kit#3]
Could not find any code related to PCF exchange app. Is it intentional? are participants of dataspace supposed to build their own PCF exchange app ?

**Comment by rishabh-rdg** [comment]
@ChristopherWinterZF  Hi Cristopher, we were asked to contact you to get more information on PCF KIT. Can you tell us if there is something planned to be uploaded in this repository? 
Also about PCF Data Exchange App - Is this something we need to build on our own? Use 3rd Party? Or there will be a sample code available for it?

**Comment by ChristopherWinterZF** [comment]
Hi @rishabh-rdg to Release a Kit the guidlines define to create an own repo for each KIT. As the PCF-KIT is based atm only on COTS application from SAP und Siemens there are no sample code available. It is Definition and Spec only. I created a Pull request as I forked the repo but then I went to my summer vacation and returned yesterday. I need to have a look why the pullrequest is sill not accepted.
